pyproject.toml

```diff
@@ -52,12 +52,13 @@ dynamic = ["version"]
 test = [
     "pytest>=7.0.0,<8.0.0",
     "pytest-cov",
+    "numtest"
```

Comment:
I am not sure we really need this dependency. All tests passed on my end *without* `numtest`, just with the `...` at proper places. Unless I am missing something, we do not need `numtest`, and by applying Occam's razor, it should be removed.